editor/import/3d/scene_import_settings.cpp

```diff
@@ -1891,6 +1891,9 @@ SceneImportSettingsDialog::SceneImportSettingsDialog() {
 	inspector = memnew(EditorInspector);
 	inspector->set_custom_minimum_size(Size2(300 * EDSCALE, 0));
 	inspector->connect(SNAME("property_edited"), callable_mp(this, &SceneImportSettingsDialog::_inspector_property_edited));
+	// Display the same tooltips as in the Import dock
+	inspector->set_object_class("ResourceImporterScene");
```

Comment:
I've tested this locally and it works:

```suggestion
	// Display the same tooltips as in the Import dock.
	inspector->set_object_class(ResourceImporterScene::get_class_static());
```

It removes the dependency on a string which is prone to typos.